core/templates/dev/head/pages/exploration_editor/ExplorationPropertyService.js

```diff
@@ -0,0 +1,97 @@
+oppia.factory('ExplorationPropertyService', [
```

Comment:
Add Copyrights. (like [here](https://github.com/Dragneel7/oppia/blob/c28f7d82bc565c2870efcda29d44e6ee232ee80f/core/templates/dev/head/pages/exploration_editor/EditorServices.js#L1))

Reply:
@Dragneel7, have you gone through this?